shell/utils/validators/formRules/index.ts

```diff
@@ -176,6 +176,17 @@ export default function(t: Translation, { key = 'Value' }: ValidationOptions): {
     return containers.map((container: any) => containerImage(container)).find((containerError: string) => containerError);
   };
 
+  const registryUrl = (privateRegistryURL: string) => {
```

Comment:
I decided to put this in shell instead of in the pkg validators because we have registries in other areas of rancher.